composer.json

```
@@ -13,11 +13,11 @@
     "require": {
         "php": "^7.2|^8",
         "vimeo/psalm": "^3.6.2 || dev-master",
-        "doctrine/collections": "^1.0",
-        "doctrine/orm": "^2.6",
         "composer/semver": "^1.4 || ^2.0 || ^3.0" 
     },
     "conflict": {
+        "doctrine/collections": "<1.0",
+        "doctrine/orm": "<2.6",
```

Shouldn't we conflict with newer version, making those the negative of the dev requirements?
```suggestion
        "doctrine/collections": "<1.0,>=2.0",
        "doctrine/orm": "<2.6,>=3.0",
```